PrebidMobile/PrebidMobileRendering/Utilities/UserAgentService.swift

```diff
@@ -0,0 +1,54 @@
+/*   Copyright 2018-2021 Prebid.org, Inc.
+ 
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+ 
+  http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  */
+
+import WebKit
+
+@objc(PBMUserAgentService) @objcMembers
+public class UserAgentService: NSObject {
+    
+    public static let shared = UserAgentService()
+    
+    public private(set) var userAgent: String = ""
+    
+    private let webView = WKWebView()
+    
+    override init() {
+        super.init()
+        
+        generateUserAgent()
+    }
+    
+    public func generateUserAgent(completion: (() -> Void)? = nil) {
```

Comment:
Let's call this method `fetchUserAgent` . 

Also looking at the signature, I expect that the UA string will be a parameter of the completion handler. I see that it is not applicable for the current usage of this method, but please consider such interface changes.